pkg/network/status/status_test.go

```diff
@@ -320,12 +334,19 @@ func TestCancelPodProbing(t *testing.T) {
 
 	// Cancel Pod probing
 	prober.CancelPodProbing(pod)
-
-	// Check that the requests were for the new version or the other Ingress
-	close(requests)
-	for req := range requests {
-		if !strings.HasPrefix(req.Host, otherDomain) && !strings.HasPrefix(req.Host, parallelDomain) {
-			t.Fatalf("Host = %s, want: %s or %s", req.Host, otherDomain, parallelDomain)
+	cancelTime := time.Now()
+
+	// Check that there are no requests for the old Ingress and the requests predate cancellation
```

Comment:
```
for req := range requests {
...
}
```

Reply:
`requests` is never closed and can't be closed on the sending side.

Reply:
then `!ok` is not possible either.

Reply:
You are right, I don't know why I though `ok` would be `false` if these was no items in the channel :weary:.
I'll use `select`.